src/explore-education-statistics-frontend/src/components/AccordionWithAnalytics.tsx

```diff
@@ -9,7 +9,9 @@ const AccordionWithAnalytics = (props: AccordionProps) => {
       onToggle={accordionSection => {
         logEvent(
           'Accordion',
-          `${accordionSection.title} accordion opened`,
+          accordionSection.publicationTitle
+            ? `${accordionSection.publicationTitle} | ${accordionSection.title} accordion opened`
+            : `${accordionSection.title} accordion opened`,
           `URL: ${window.location.pathname} | Accordion ID: ${accordionSection.id}`,
```

Comment:
Does Accordion ID add anything here? Looks like the value is `Extra-Information-Sections-3-Heading` which is pretty useless in GA. If you remove this and the URL then it would free up the label parameter of the event to be used for the Publication Title which would be an appropriate use of it since it's meant for categorising the events.

As it currently stands there will be a lot of different "[Publication Title] Contact Us Accordion Opened" event actions diluting the total set of different event actions that we have and no way of filtering just them.

I think it would be better to keep the Event action as "Contact Us Accordion Opened" and then use the label param for the Publication Title.

The GA user can then drill into the "Contact Us Accordion Opened" and see a breakdown by Publication Title.